doc/api/fs.md

```diff
@@ -6825,28 +6825,33 @@ The following constants are meant for use as the `mode` parameter passed to
 <table>
   <tr>
     <th>Constant</th>
+    <th>Binary</th>
     <th>Description</th>
   </tr>
   <tr>
     <td><code>F_OK</code></td>
+    <td><code>~~0</code></td>
     <td>Flag indicating that the file is visible to the calling process.
      This is useful for determining if a file exists, but says nothing
      about <code>rwx</code> permissions. Default if no mode is specified.</td>
   </tr>
   <tr>
-    <td><code>R_OK</code></td>
-    <td>Flag indicating that the file can be read by the calling process.</td>
+    <td><code>X_OK</code></td>
+    <td><code>~~1</code></td>
```

Comment:
The `~` character is almost exclusively used as the unary bitwise complement operator (in JavaScript, C, C++, Java, ...), which is extremely confusing here.